javaparser-core/src/main/java/com/github/javaparser/ast/expr/Expression.java

```diff
@@ -881,4 +877,19 @@ public final boolean elidesTypeArguments() {
         NodeWithTypeArguments nwta = (NodeWithTypeArguments) this;
         return scope.elidesTypeArguments() && (!nwta.getTypeArguments().isPresent() || nwta.isUsingDiamondOperator());
     }
+
```

Comment:
This appears to be generated by the TypeCastingGenerator, whereas it seems to have been added manually.

Reply:
I think the same applies as the comment I left below regarding formatting and project configuration. I think this is generated formatting which just hasn't been included before.